cirq-core/cirq/ops/measure_util.py

```diff
@@ -33,19 +33,28 @@ def measure_single_paulistring(
     pauli_observable: pauli_string.PauliString,
     key: Optional[Union[str, 'cirq.MeasurementKey']] = None,
 ) -> raw_types.Operation:
-    """Returns a single PauliMeasurementGate which measures the pauli observable
+    """Returns an operation, from `cirq.PauliMeasurementGate`, to measure the `pauli_observable`.
+
+    Note that the coefficient of `pauli_observable` will be ignored and the resulting operation
+    will measure the single pauli observable with coefficient 1. For example:
```

Comment:
ugh.  Shouldn't this yell.  If I pass in `-cirq.X(q[0]) * cirq.Z(q[1])` it silently failing seems really counter intuitive.  Maybe we should open an issue on this?

Reply:
Done. https://github.com/quantumlib/Cirq/issues/5615